diff --git a/controllers.py b/controllers.py
index 79cb28a..00b4da6 100644
--- a/controllers.py
+++ b/controllers.py
@@ -38,6 +38,7 @@ def index(param=None):
    ctrl_info= "ctrl: index, view: index.html"
    if not param is None:
        print (param)
+   from .tools import formInfoThroughCookies; formInfoThroughCookies()
    return locals()
 
 
@@ -145,4 +146,3 @@ def api(tablename, rec_id=None):
 #  http -f POST localhost:8000/simile_fs/api/test_table/ f0=111111 f1=2222222 f2=333333
 #  http -f DELETE localhost:8000/simile_fs/api/test_table/2
 #  http -f PUT localhost:8000/simile_fs/api/test_table/2 f0=111111 f1=2222222 f2=333333
-
